README.md

```diff
@@ -15,8 +15,8 @@ It is responsible for rendering the UI for the landing page after the user logs
 
 ## Contributing / Development
 
-[Setup local development environment for OpenMRS SPA](https://wiki.openmrs.org/display/projects/Setup+local+development+environment+for+OpenMRS+SPA)
+[Setup local development environment for OpenMRS SPA](https://wiki.openmrs.org/display/projects/Setup+local+development+environment+for+OpenMRS+SPA )
```

Comment:
@denniskigen could you be aware of the correct link to the local development set up instruction? The existing one link is pointing to a non public content.

Reply:
https://o3-docs.openmrs.org/docs/frontend-modules/development